swan-lake/learn/using-the-cli-tools.md

```diff
@@ -53,58 +54,58 @@ BALLERINA COMMANDS
    Core Commands:
         build           Compile Ballerina program into an executable
         run             Build and run Ballerina program
-        test            Run module tests
+        test            Run package tests
         doc             Generate API documentation
         clean           Clean artifacts generated during the build
         format          Format Ballerina sources
 
-   Module Commands:
-        pull            Pull a module from Ballerina Central
-        push            Upload module to the Ballerina Central
-        search          Search Ballerina Central for modules
-
-   Project Commands:
-        new             Create a new Ballerina project
-        add             Create a new Ballerina module in a project
+   Package Commands:
+        new             Create a new Ballerina package
+        init            Create a new Ballerina package in the current directory
+        add             Create a new Ballerina module in the package
+        pull            Pull a package from Ballerina Central
+        push            Upload a package to Ballerina Central
+        search          Search Ballerina Central for packages
 
    Other Commands:
         encrypt         Encrypt sensitive data
         grpc            Generate Ballerina sources for the given protobuf definition
         openapi         Generate Ballerina sources for the given OpenAPI definition and vice versa.
         version         Print Ballerina version
         bindgen         Generate Ballerina bindings for Java APIs
+        shell           Run ballerina interactive REPL [EXPERIMENTAL]
 
    Update Commands:
         dist            Manage Ballerina distributions
-        update          Update the Ballerina Tool
+        update          Update the Ballerina tool
```

Comment:
Shall we keep it as it is? @praneesha What do you think?

Reply:
I copied the output of the command and pasted it there actually. So the `bal --help` has this format in the description. 

I copied it since i thought it would look consistent

Reply:
Need to keep "Tool" her and update the CLI output also later.

Reply:
ack. Will change to caps